src/io/state-file/index.ts

```diff
@@ -35,7 +36,8 @@ export async function save(fileName: string) {
   const zip = new JSZip();
   const manifest: Manifest = {
     version: VERSION,
-    dataSets: [],
+    datasets: [],
+    remoteDatasetFileEntries: {},
```

Comment:
I think the suffix `Entries` is redundant. We can leave it as `remoteDatasetFiles`.

Reply:
Agreed.  Cut to `remoteFiles` to avoid overlap with `RemoteDatasetFile` in `datasets-files.ts`